LocalSettings.php

```diff
@@ -138,9 +138,7 @@
  * redirect loops when "pretty URLs" are used.
  * @var bool $wgUsePathInfo
  */
-$wgUsePathInfo = ( strpos(php_sapi_name(), 'cgi') === false ) &&
-        ( strpos(php_sapi_name(), 'apache2filter') === false ) &&
-        ( strpos(php_sapi_name(), 'isapi') === false );
+$wgUsePathInfo = ( php_sapi_name() == 'apache2handler' ) || ( php_sapi_name() == 'fpm-fcgi' ); # Wikia change - SUS-5825
```

Comment:
How does `apache2handler` replace the previous values?

Reply:
`apache2handler` is the sapi used when PHP is run as a Apache module.

Reply:
You don't have to mark it as Wikia change, since LocalSettings is supposed to be the custom file.